content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
+
+h4. Key benefits
+
+* Cost efficiency.
+* Reduced system load.
+* Message rate limit mitigation.
+
+h4. Use cases
+
+The are the primary use cases where server-side batching provides significant value:
+
+* *High-volume chat applications:*
+A live event chat platform experiences large spikes in message volumes during peak moments, such as live sports games or concerts. Managing high-frequency chat messages and reactions from thousands of users simultaneously can increase costs and the risk of hitting message rate limits.
+_*Solution:*_ Implementing server-side batching allows the platform to group multiple chat messages and reactions into single batches before delivering them to subscribers. This reduces the number of individual messages processed and helps maintain a seamless and interactive user experience during high-activity periods.
+
+* *Fan engagement platforms:*
+A sports fan engagement app allows users to send reactions, such as likes and comments, during live matches. During key moments, like a goal being scored, the app faces a surge in user interactions, leading to a spike in message volume.
+_*Solution:*_ Server-side batching can combine multiple reactions into a batch before sending them to subscribers. This helps manage the high volume of interactions efficiently, preventing rate limits from being exceeded and ensuring all reactions are delivered promptly.
+
+* *Cursor tracking:*
+A collaborative workspace application tracks user presence and cursor movements in realtime, showing the live cursor positions of multiple users. Continuous updates on cursor positions from numerous users can generate a high volume of messages, increasing costs and potentially hitting rate limits.
+_*Solution:*_ The application can batch these frequent updates into single messages by applying server-side batching to the cursor and presence messages. This reduces the number of messages processed and sent, lowering costs and maintaining realtime tracking accuracy.
+
+h4. How server-side batching works
+
+The server-side batching process works through a *batching policy* and a *batching interval:*
+
+h4. Batching policy
+
+The batching policy defines the rules and behaviors for aggregating messages during the batching interval. The initial release supports the *simple* policy, which groups multiple incoming messages into a single batch without altering their content or metadata.
+
+A batch can contain up to 50 messages by count or total data size. For example, if you have 60 messages, they will be split into two batches: one with 50 messages and another with 10. If the combined data size of 50 messages exceeds a specific bandwidth threshold _e.g. in kilobytes_ the batch size will be limited to that threshold.
```

Comment:
Rather than 'a specific bandwidth threshold' we count each message as 2KiB (not Kb), so I'm presuming it will be if the batch exceeds 100KiB. Is that correct @JoaoDiasAbly ?

Reply:
> Rather than 'a specific bandwidth threshold' we count each message as 2KiB 
This is for billing

Messages can contain fewer data than that. We've actually changed the messages limit in a batch from 50 to 200 yesterday. The bandwidth size is still what was configured before, 65536 bytes, which I don't think we want to advertise. So I'd change the 50 to 200 but other than that I think it's okay